static/app/views/organizationGroupDetails/groupEventDetails/index.tsx

```diff
@@ -31,51 +32,34 @@ export interface GroupEventDetailsProps
   selection: PageFilters;
 }
 
-type State = typeof OrganizationEnvironmentsStore['state'];
+export function GroupEventDetailsContainer(props: GroupEventDetailsProps) {
+  const state = useLegacyStore(OrganizationEnvironmentsStore);
```

Comment:
minor nit, but I might destructure out `environments` and `error` here, since the component is short

Reply:
I'm not a fan of destructuring props because in majority of the cases all it does it ends up hiding the source of the value/reference. Even if the component is small, maybe someone is going to add some functionality later and I'd rather keep it to a bare minimum